packages/core/contracts/optimistic-assertor/interfaces/OptimisticAssertorInterface.sol

```diff
@@ -20,4 +20,26 @@ interface OptimisticAssertorInterface {
         uint256 assertionTime; // Time of the assertion.
         uint256 expirationTime;
     }
+
+    event AssertionMade(
+        bytes32 assertionId,
+        bytes claim,
+        address indexed proposer,
+        address indexed callbackRecipient,
+        address indexed sovereignSecurityManager,
+        IERC20 currency,
+        uint256 bond,
+        uint256 liveness
```

Comment:
It might be easier for off-chain observers to have expirationTime instead of liveness similar to what OO does.